casr/src/util.rs

```diff
@@ -32,19 +32,8 @@ use which::which;
 ///
 /// * `argv` - executable file options
 pub fn call_sub_tool(matches: &ArgMatches, argv: &[&str], name: &str) -> Result<()> {
```

Comment:
I think, this function call only casr-san. It should be renamed.

Reply:
done